### PR TITLE
Links Required Metadata and Files in Viz Widget to Tabs

### PR DESCRIPTION
**Pull request by mtribone**
Moves text into locale for translation, removes icons from tabs, and links the requirements to the correct section of the tabs.

### CODE REVIEW

**Comment by mtribone** [comment]
screen shot 2017-03-07 at 11 47 11 am

**Comment by mtribone** [comment]
screen shot 2017-03-07 at 11 42 14 am

File: app/views/curation_concerns/base/_form_progress.html.erb [hunk 1]
```diff
@@ -5,11 +5,15 @@
   <div class="list-group">
     <div class="list-group-item">
       <fieldset>
-        <legend class="legend-save-work">Requirements</legend>
+        <legend class="legend-save-work"><%= t("curation_concerns.base.form_progress.legend_required") %></legend>
         <ul class="requirements">
-          <li class="incomplete" id="required-metadata">Enter required metadata</li>
+          <li class="incomplete" id="required-metadata">
+            <a href="#metadata"><%= t("curation_concerns.base.form_progress.metadata_required") %></a>
```
Comment (awead) [inline app/views/curation_concerns/base/_form_progress.html.erb:11]: @mtribone could there be a line somewhere in feature or view test that checks for the presence of this link? Just add something to an existing test.

File: app/views/curation_concerns/base/_form_progress.html.erb [hunk 2]
```diff
@@ -5,11 +5,15 @@
...
+          </li>
           <% if Sufia.config.work_requires_files %>
-            <li class="incomplete" id="required-files">Add files</li>
+            <li class="incomplete" id="required-files">
+              <a href="#files"><%= t("curation_concerns.base.form_progress.files_required") %></a>
```
Comment (awead) [inline app/views/curation_concerns/base/_form_progress.html.erb:15]: same here.

**Review by awead — CHANGES_REQUESTED** [review]

File: spec/features/generic_work/upload_and_delete_spec.rb [hunk 3]
```diff
@@ -64,6 +64,8 @@
         expect(page).to have_no_css("#generic_work_contributor")
 
         within("#savewidget") do
+          expect(page).to have_selector("a[href='#metadata']")
```
Comment (mtribone) [inline spec/features/generic_work/upload_and_delete_spec.rb:67]: Here they be @awead ↕️

**Review by awead — APPROVED** [review]

**Pull request merged**